styleguide/source/_patterns/03-organisms/four-up-teaser.json

```diff
@@ -0,0 +1,70 @@
+{
+  "items": [
+    {
+      "patternLab": true,
+      "link": {
+        "title": "Link to something",
+        "href": "#",
+        "text": "Lorem ipsum dolor sit amet.",
+        "class": "ama__link-black",
+        "target": "_self"
+      },
+      "image": {
+        "alt": "alt text",
+        "src": "https://ipsumimage.appspot.com/800x600x186?l=3:2|800x600&s=36",
+        "height": "600",
+        "width": "800"
+      },
+      "date": "Jan 19, 2018",
+      "readtime": "4 min read"
+    },
+    {
+      "patternLab": true,
+      "link": {
+        "title": "Link to something",
+        "href": "#",
+        "text": "Lorem ipsum dolor sit amet.",
+        "class": "ama__link-black",
+        "target": "_self"
+      },
+      "image": {
+        "alt": "alt text",
+        "src": "https://ipsumimage.appspot.com/800x600x186?l=3:2|800x600&s=36",
+        "height": "600",
+        "width": "800"
+      }
+    },
+    {
+      "patternLab": true,
+      "link": {
+        "title": "Link to something",
+        "href": "#",
+        "text": "Lorem ipsum dolor sit amet.",
+        "class": "ama__link-black",
+        "target": "_self"
+      },
+      "image": {
+        "alt": "alt text",
+        "src": "https://ipsumimage.appspot.com/800x600x186?l=3:2|800x600&s=36",
+        "height": "600",
+        "width": "800"
+      }
+    },
+    {
+      "patternLab": true,
+      "link": {
+        "title": "Link to something",
+        "href": "#",
+        "text": "Lorem ipsum dolor sit amet.",
+        "class": "ama__link-black",
```

Comment:
This should be `ama__link--black` since black is a modifier. Also I think black is the default so you may not need the class at all.